src/tests/multihost/alltests/test_services.py

```diff
@@ -182,6 +182,8 @@ def test_0006_bz1909755(self, multihost, backupsssdconf):
           2. Should succeed
           3. Should succeed
         """
+        if not multihost.client[0].detect_files_provider():
+            pytest.skip("Files Provider support isn't available, skipping")
```

Comment:
According to the description, this test isn't specific to 'files provider' functionality.

Reply:
The fact that test uses 'files provider' does not mean it tests 'files provider'.
Logic of this test does not depend on any specific provider being used.
It can be proxy, ldap or anything.
Now, I do not know if this tests actually makes sense (imo not)
But it doesn't make sense to condition this tests on availability of files provider.